config.yaml.defaults

```diff
@@ -70,6 +70,163 @@ app:
       icon: Info
       url: /about
 
+  homepage_layouts:
```

Comment:
I think it makes sense to establish the defaults for each widget, so that we can add extra parameters. Most important is a `show` toggle (since not all SP instances will want to show all these widgets):
```yaml
homepage_layouts:
  sourceCounts:
      show: true
      days_past_to_include: 7
      resizeable: false
      layout: 
         xlg: [14, 0, 2, 1]
         lg: ...
```